src/main/java/com/salesforce/dynamodbv2/mt/mappers/metadata/PrimaryKey.java

```diff
@@ -77,7 +77,8 @@ public String toString() {
         return "{"
             + "hashKey='" + hashKey + '\''
             + ", hashKeyType=" + hashKeyType
-            + (rangeKey.map(s -> ", rangeKey=" + s + ", rangeKeyType=" + rangeKeyType.get()).orElse(""))
+            + (rangeKey.map(s -> ", rangeKey=" + s + ", rangeKeyType="
+            + (rangeKeyType.map(Enum::name).orElse("null"))))
```

Comment:
why `name` over `toString`? and should that be `"null"` or something else (signifying none--e.g., `"<NONE>"`)?

Reply:
addressed in dc216a3107b51b4390c250e5d718fcb1e65d8993